### PR TITLE
SAML support

### PR DESCRIPTION
**Pull request by rart**

### CODE REVIEW

File: ui/app/src/models/User.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
```
Comment (sumerjabri) [inline ui/app/src/models/User.ts]: Fix copyright notice.

File: ui/app/src/services/auth.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
```
Comment (sumerjabri) [inline ui/app/src/services/auth.ts]: Fix copyright notice.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: ui/app/src/models/User.ts (hunk 1 above)
Comment (rart) [reply ui/app/src/models/User.ts]: Done

File: ui/app/src/services/auth.ts (hunk 2 above)
Comment (rart) [reply ui/app/src/services/auth.ts]: Done

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**